### ISSUE TITLE
RTD automatic building when merging to main

### ISSUE THREAD

**Issue by HTenkanen** [r5py/r5py#47]
@christophfink : I guess at the moment, the RTD does not automatically build the docs when something is merged to main? I think we could enable that? 

Could you btw. add me to the RTD project as a maintainer (my RTD-username is `htenkanen`)? --> Admin --> Maintainers --> Add Maintainer

**Comment by christophfink** [comment]
It should build automatically, and I’ll add you :)

**Comment by HTenkanen** [comment]
Updated, the default RTD view to /latest now from the settings

**Comment by christophfink** [comment]
fine with that, let’s keep a note, though, to point the default to stable, again (at latest, when we push packages to pypi and conda-forge).

**Comment by HTenkanen** [comment]
Yes, I think once we are ready to publish, we could "freeze" the docs at that point and point to /stable as the default one. Then the latest could serve as the dev-docs. Now everything is under dev so there's no difference really 🙂

**Comment by HTenkanen** [comment]
Added a note about this

**Comment by christophfink** [comment]
I have been turning PR building on and off, before releases
